### DISCUSSION TITLE
Delete this one

### DISCUSSION THREAD

**Discussion by shan-github** [adobe/react-spectrum#2863] [General]
Currently in React spectrum I can't access **Event object** in button's `onPress` prop and there are other component that require this functionality. Can this be implemented?

**Comment by reidbarber** [comment]
onPress should pass a PressEvent:
https://github.com/adobe/react-spectrum/blob/404d41859b7d6f56201d7fc01bd9f22ae3512937/packages/%40react-types/shared/src/events.d.ts#L31-L46

If you're looking for something else, can you explain your use case?
